frontend/gateway/forwarder/forward.go

```diff
@@ -22,11 +25,10 @@ import (
 	digest "github.com/opencontainers/go-digest"
 	"github.com/pkg/errors"
 	fstypes "github.com/tonistiigi/fsutil/types"
-	"golang.org/x/sync/errgroup"
 )
 
-func llbBridgeToGatewayClient(ctx context.Context, llbBridge frontend.FrontendLLBBridge, opts map[string]string, inputs map[string]*opspb.Definition, w worker.Infos, sid string, sm *session.Manager) (*bridgeClient, error) {
-	bc := &bridgeClient{
+func LLBBridgeToGatewayClient(ctx context.Context, llbBridge frontend.FrontendLLBBridge, exec executor.Executor, opts map[string]string, inputs map[string]*opspb.Definition, w worker.Infos, sid string, sm *session.Manager) (*BridgeClient, error) {
+	bc := &BridgeClient{
```

Comment:
Why were `func llbBridgeToGatewayClient` and `type bridgeClient` renamed? I don't see why they have to be exported. ("They were renamed in v0.12" is not a valid reason.)

Reply:
`bridgeClient` is made exported since exported func LLBBridgeToGatewayClient should have exported-return
` > [stage-0 5/6] RUN --mount=target=/go/src/github.com/moby/buildkit --mount=target=/root/.cache,type=cache   GOARCH=amd64 golangci-lint run &&   GOARCH=arm64 golangci-lint run:                                                            
2.809 frontend/gateway/forwarder/forward.go:29:225: unexported-return: exported func LLBBridgeToGatewayClient returns unexported type *forwarder.bridgeClient, which can be annoying to use (revive)                                         
2.809 func LLBBridgeToGatewayClient(ctx context.Context, llbBridge frontend.FrontendLLBBridge, exec executor.Executor, opts map[string]string, inputs map[string]*opspb.Definition, w worker.Infos, sid string, sm *session.Manager) (*bridgeClient, error) {
`

Reply:
Don't change `func llbBridgeToGatewayClient` to be exported, then!